src/raftstore/store/store.rs

```diff
@@ -2616,17 +2613,16 @@ impl<T: Transport, C: PdClient> Store<T, C> {
             .with_label_values(&["receiving"])
             .set(snap_stats.receiving_count as i64);
 
-        let mut apply_snapshot_count = 0;
-        for peer in self.region_peers.values_mut() {
-            if peer.mut_store().check_applying_snap() {
-                apply_snapshot_count += 1;
-            }
-        }
+        let apply_snapshot_count = self
```

Comment:
What about record the number of snapshots in a global counter? So It does not need to check all peers.

Reply:
How about do it in another PR? The PR is already a little huge.